apps/meteor/app/settings/server/functions/convertValue.ts

```diff
@@ -10,5 +10,8 @@ export const convertValue = (value: 'true' | 'false' | string, type: ISetting['t
 	if (type === 'int') {
 		return parseInt(value);
 	}
+	if (type === 'multiSelect') {
+		return JSON.parse(value);
```

Comment:
should we add something to prevent parsing errors here? what an invalid JSON should do? break the application? be ignored?